dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageProblem.java

```diff
@@ -60,7 +60,10 @@
    *     not exist in the class path.
    */
   LinkageProblem(
-      String symbolProblemMessage, ClassFile sourceClass, Symbol symbol, ClassFile targetClass) {
+      String symbolProblemMessage,
+      ClassFile sourceClass,
+      Symbol symbol,
+      @Nullable ClassFile targetClass) {
```

Comment:
I don't think there's a test for the case where this is null yet.

Reply:
Let me add a test.

Reply:
Added a test.